src/openqaoa-core/README.md

```diff
@@ -5,73 +5,112 @@
+
 </div>
 
 # OpenQAOA-Core
 
-OpenQAOA is a multi-backend python library for quantum optimization using QAOA on Quantum computers and Quantum computer simulators. This package is part of a set of OpenQAOA plug-ins and forms the core functionality of the package. It includes all backend agnostic elements of the QAOA process. Along with other OpenQAOA plug-in extensions enables users to seemlessly run QAOA computations across multiple Quantum Computers. Additionally, it also comes bundled with Entropica's fast QAOA simulator called `vectorized` and a unit-depth QAOA analytical simulator (you can find more details in the Getting Started section). Check out OpenQAOA website [https://openqaoa.entropicalabs.com/](https://openqaoa.entropicalabs.com/)
+OpenQAOA is a multi-backend python library for quantum optimization using QAOA
+on Quantum computers and Quantum computer simulators. This package is part of a
+set of OpenQAOA plug-ins and forms the core functionality of the package. It
+includes all backend agnostic elements of the QAOA process. Along with other
+OpenQAOA plug-in extensions enables users to seemlessly run QAOA computations
+across multiple Quantum Computers. Additionally, it also comes bundled with
+Entropica's fast QAOA simulator called `vectorized` and a unit-depth QAOA
+analytical simulator (you can find more details in the Getting Started section).
+Check out OpenQAOA website
+[https://openqaoa.entropicalabs.com/](https://openqaoa.entropicalabs.com/)
 
 **OpenQAOA is currently in OpenBeta.**
 
-Please, consider [joining our discord](https://discord.gg/ana76wkKBd) if you want to be part of our community and participate in the OpenQAOA's development. 
+Please, consider [joining our discord](https://discord.gg/ana76wkKBd) if you
+want to be part of our community and participate in the OpenQAOA's development.
 
 ## Installation instructions
 
 ### Install via PyPI
 
-You can install the latest version of openqaoa-core directly from PyPi. We recommend creating a virtual environment with `python>=3.8` first and then simply pip install openqaoa-core with the following command.
+You can install the latest version of openqaoa-core directly from PyPi. We
+recommend creating a virtual environment with `python>=3.9` first and then
+simply pip install openqaoa-core with the following command.
 
 ```bash
 pip install openqaoa-core
 ```
 
 ### Installation instructions for Developers
-OpenQAOA-Core does not yet support developer install as a standalone package. If you wish to work in developer mode, please install the entire library. Instructions are available [here]()
 
-Should you face any issue during the installation, please drop us an email at openqaoa@entropicalabs.com or open an issue!
+OpenQAOA-Core does not yet support developer install as a standalone package. If
+you wish to work in developer mode, please install the entire library.
+Instructions are available [here]()
```

Comment:
Can you please add the missing link? https://github.com/entropicalabs/openqaoa/blob/dev/README.md